examples/form-styles.css

```diff
@@ -0,0 +1,3 @@
+.schemaform-expandUnder-indent{
+  margin-left: 29px;
```

Comment:
I think there should be classes for this [here](https://design.va.gov/foundation/utilities/margins)